src/terminal_sandbox.ml

```diff
@@ -69,23 +69,30 @@ let get_shell_args () =
 
 type t = Terminal.t
 
-let create sandbox =
+let create ?name ?command sandbox =
   let shell_path = get_shell_path () in
   let shell_args = get_shell_args () in
   let ({ Cmd.bin; args } as command) =
-    match Sandbox.get_command sandbox shell_path shell_args with
-    | Spawn spawn -> spawn
-    | Shell command_line -> (
-      match Platform.shell with
-      | Sh bin -> { bin; args = [ "-c"; command_line ] }
-      | PowerShell bin -> { bin; args = [ "-c"; "& " ^ command_line ] } )
+    Option.value command
```

Comment:
Use a pattern match here instead of `Option.value`. It's not obvious that whatever is generating `~default` won't fail, so let's just be safe and only call it when needed.